src/common/src/texpr.c

```diff
@@ -2041,13 +2061,20 @@ void vectorMathFunc(int16_t functionId, tExprOperandInfo *pInputs, int32_t numIn
     if (!hasNullInputs) {
       switch (functionId) {
         case TSDB_FUNC_SCALAR_LOG: {
-          assert(numInputs == 2);
-          double base = 0;
-          GET_TYPED_DATA(base, double, pInputs[1].type, inputData[1]);
+          assert(numInputs == 1 || numInputs == 2);
```

Comment:
I guess this line is useless, because the error has already been caught at line 1202

Reply:
by the way, we should NOT use assert() in product code. There have been already many asserts, let's at least not add new ones.

Reply:
yes. I will remove it.